### PR TITLE
Chore: Added tests for canned_responses

### PR DESCRIPTION
**Pull request by venarius**
# Pull Request Template

## Description

This PR adds tests for the canned_responses_controller.rb

Improves #37 

## Type of change
- [x] Chore

## How Has This Been Tested?

Checked that all tests went through, that Rubocop is satisfied and verified my additions with simplecov locally.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: spec/controllers/api/v1/canned_responses_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    context 'when it is an authenticated user' do
+      let(:agent) { create(:user, account: account, role: :agent) }
+
+      it 'returns all the canned responses' do
+        get '/api/v1/canned_responses',
+            headers: agent.create_new_auth_token,
+            as: :json
+
+        expect(response).to have_http_status(:success)
+        expect(JSON.parse(response.body)).to eq([CannedResponse.last].as_json)
```
Comment (pranavrajs) [inline spec/controllers/api/v1/canned_responses_controller_spec.rb]: I guess `account.canned_responses` would be a better approach than `CannedResponse`

**Comment by pranavrajs** [comment]
@venarius Thanks for the PR, Great work! ❤️ I just have a small comment, please see the inline comment.

File: spec/controllers/api/v1/canned_responses_controller_spec.rb (hunk 1 above)
Comment (venarius) [reply spec/controllers/api/v1/canned_responses_controller_spec.rb]: Yes, that is true. Will change it :)

**Comment by venarius** [comment]
@pranavrajs done :)

**Pull request merged**